web/app.js

```diff
@@ -87,7 +87,7 @@ var DefaultExternalServices = {
   },
 };
 
-var PDFViewerApplication = {
+let PDFViewerApplication = {
```

Comment:
Please note that using `let` here won't change the ability to access `PDFViewerApplication` through the console (or similar), since [viewer.js#L183](https://github.com/mozilla/pdf.js/blob/d6a6f390ae9a39554b73a942860f67badbef1b4b/web/viewer.js#L183) and [viewer.js#L187](https://github.com/mozilla/pdf.js/blob/d6a6f390ae9a39554b73a942860f67badbef1b4b/web/viewer.js#L187) already ensures that it's defined globally.